### PR TITLE
Add Single Replica Mode lane for SSP

### PR DESCRIPTION
**Pull request by borod108**
Add a lane to test SSP works properly in SNO mode

Signed-off-by: borod108 <boris.od@gmail.com>

### CODE REVIEW

**Comment by akrejcir** [comment]
Let's wait until the PR that uses it is merged:
https://github.com/kubevirt/ssp-operator/pull/262

/hold

**Comment by ksimon1** [comment]
@vrutkovs can you please review this PR?

File: ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-master.yaml [hunk 1]
```diff
@@ -99,6 +99,28 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: ipi-gcp
+- as: sno-e2e-functests
```
Comment (vrutkovs) [inline ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-master.yaml:102]:
```suggestion
- as: e2e-single-node-functests
```

Comment (borod108) [reply ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-master.yaml]: done

**Review by vrutkovs — APPROVED** [review]
/approve
/lgtm

**Pull request merged**